src/main.js

```diff
@@ -227,12 +228,30 @@ app.on('ready', function() {
 
     trayIcon.setToolTip(app.getName());
     trayIcon.on('click', function() {
+      if (process.platform === 'win32') {
+        if (config.minimizeToTray) {
+          if (mainWindow.isHidden) {
```

Comment:
Yes this was intended as a toggle. As currently the tray icon has no 'use' for the left click, if the window is open, I find it very convenient to click again to close the window. 

Especially if I just want to check the currently open channel (I use it currently mainly for Build-Server-Notifications), I just click on the Icon, and if there is nothing that much interesting I just click again to close. Else I  have to go explicitely on the X-Button or the Minimize-Button of the window and that would be much more way with the mouse.